header/partials/subnav/template-new.html

```diff
@@ -30,8 +30,8 @@
 
 				</div>
 			</div>
-			<button class="o-header__subnav-button o-header__subnav-button--left" aria-hidden="true" disabled></button>
-			<button class="o-header__subnav-button o-header__subnav-button--right" aria-hidden="true" disabled></button>
+			<button title="subnav-spacing-btn-left" class="o-header__subnav-button o-header__subnav-button--left" aria-hidden="true" disabled></button>
```

Comment:
this looks like the value belongs in the class? also use aria-label and 'scroll left' text